Dockerfile

```diff
@@ -1,15 +1,30 @@
+ARG WALLABAG_VERSION=2.6.2
+
 FROM golang:alpine as builder
 
 # envsubst from gettext can not replace env vars with default values
 # this package is not available for ARM32 and we have to build it from source code
 # flag -ldflags "-s -w" produces a smaller executable
 RUN go install -ldflags "-s -w" -v github.com/a8m/envsubst/cmd/envsubst@v1.3.0
 
+FROM node:16-alpine3.18 as assets
+
+ARG WALLABAG_VERSION
```

Comment:
Shouldn't this be removed as you added it on line 1?

Reply:
no, every stage must explicitly define the ARGs it needs. here the one line 1 is there to define the default value to be used by all stages so it's defined once.

here the doc: https://docs.docker.com/engine/reference/builder/#scope
![image](https://github.com/wallabag/docker/assets/1480128/10b97c50-ff9d-4068-aa56-8e7982d9ad06)